src/stp/new_plays/TimeOut.cpp

```diff
@@ -29,7 +29,7 @@ uint8_t TimeOut::score(world_new::World* world) noexcept { return 100; }
 
 void TimeOut::calculateInfoForRoles() noexcept {
     const auto xPosition = -4 * control_constants::ROBOT_RADIUS;
-    const auto yPosition = Constants::STD_TIMEOUT_TO_TOP() ? field.getFieldWidth() / 2 : -field.getFieldWidth() / 2;
+    const auto yPosition = Constants::STD_TIMEOUT_TO_TOP() ? field.getFieldWidth() / 2 - 0.15 : -field.getFieldWidth() / 2 + 0.15;
```

Comment:
You are basically hard coding the coordinate system if you use this and it is a type of code cloning. Use field.getTopMostY() and field.getBottomMostY() instead.